### PR TITLE
Fixed undefined var

### PR DESCRIPTION
**Pull request by elimn**
Ticket: https://central.tri.be/issues/68765

### CODE REVIEW

File: output/gigpress_sidebar.php [hunk 1]
```diff
@@ -202,6 +202,7 @@ function gigpress_sidebar($filter = null) {
 	$further_where = '';
 
 	// Variables we need for conditionals
+	$some_results = FALSE;
```
Comment (bordoni) [inline output/gigpress_sidebar.php]: Go with `false` in lowercase.

File: output/gigpress_shows.php [hunk 2]
```diff
@@ -17,6 +17,7 @@ function gigpress_shows($filter = null, $content = null) {
 
 	global $wpdb, $gpo;
 	$further_where = $limit = '';
+	$some_results = FALSE;
```
Comment (bordoni) [inline output/gigpress_shows.php]: Go with `false` in lowercase.

Comment (elimn) [inline output/gigpress_shows.php]: @bordoni I normally would but the entire page is using the uppercase standard. Have we decided what we are going to do with GigPress, as it doesn't use the MT standard? Are we going to try and convert the whole thing?

Comment (bordoni) [inline output/gigpress_shows.php]: Slowly convert to our standards.

Comment (elimn) [inline output/gigpress_shows.php]: 👍 In that case, I'll just auto format the whole file.

**Comment by barryhughes** [comment]
Reapplying 'enhance' label - still needs one of the changes Gustavo flagged

**Pull request merged**